modules/afsocket/afsocket-source.c

```diff
@@ -397,12 +425,19 @@ afsocket_sd_process_connection(AFSocketSourceDriver *self, GSockAddr *client_add
 
 #endif
 
-  if (self->num_connections >= self->max_connections)
+  if (_connections_count_get(self) >= self->max_connections)
     {
+      gchar *source_name = cfg_tree_get_rule_name(&self->super.super.super.cfg->tree, ENC_SOURCE,
```

Comment:
Alternatively, you can use `self->super.super.group`, which is exactly the same, but you don't have to bother with the deallocation, etc.:

```
gchar *group_name = self->super.super.group;
evt_tag_str("group_name", group_name),
```